src/client/styles/components/_sidebar.scss

```diff
@@ -1,13 +1,18 @@
 .sidebar {
   a.back-link {
+    text-decoration: none;
+    display: none;
     background: #fff;
     color: #000;
-    display: block;
     margin: 0 0 20px;
 
     svg {
       float: left;
       margin: -2px 0 0;
     }
+
+    @include min-screen(966px) {
```

Comment:
Instead of hardoing the pixel value, use the variable in the toolkit `$mobile-breakpoint: 965px;` -- Ricardo clarified this for me earlier.

https://github.com/NYPL/design-toolkit/blob/master/sass/_measurements.scss